### PR TITLE
feat: timezone support via get time offset

### PR DESCRIPTION
**Pull request by Chandru-HM**
This PR is for ticket https://github.com/awslabs/iot-app-kit/issues/2663 and added the timezone support via get time offset.

## Verifying Changes
https://github.com/awslabs/iot-app-kit/assets/142866907/53db3ac3-e5cf-4b44-a69b-d177413b1b6d

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: packages/react-components/src/components/kpi/kpiBase.tsx [hunk 1]
```diff
@@ -129,7 +130,14 @@ export const KpiBase: React.FC<KPIBaseProperties> = ({
                   }}
                 />
                 <div className='timestamp' data-testid='kpi-timestamp'>
-                  {isLoading ? '-' : new Date(point.x).toLocaleString()}
+                  {isLoading ? (
+                    '-'
+                  ) : (
+                    <DateTime
+                      dateTime={point.x}
+                      pattern='dd/MM/yyyy, HH:mm:ss'
```
Comment (jmbuss) [inline packages/react-components/src/components/kpi/kpiBase.tsx:138]: Can we make these constants? I believe this exists already here - packages/react-components/src/utils/time.ts

Comment (jmbuss) [reply packages/react-components/src/components/kpi/kpiBase.tsx:138]: Also, let's default the DateTime componeent to this so we don't need to pass it everywhere.

File: packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/modeledDataStreamQueryEditor/assetExplorer/assetTable/assetTableColumnDefinitionsFactory.tsx [hunk 2]
```diff
@@ -87,7 +87,14 @@ export class AssetTableColumnDefinitionsFactory {
       id: 'creationDate',
       header: 'Creation Date',
       cell: ({ creationDate }) =>
-        creationDate ? getFormattedDateTime(creationDate) : '-',
+        creationDate ? (
+          <DateTime
+            dateTime={creationDate.getTime()}
```
Comment (jmbuss) [inline packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/modeledDataStreamQueryEditor/assetExplorer/assetTable/assetTableColumnDefinitionsFactory.tsx:92]: Is it possible to change creationDate to a number prop? I don't think we should do a bunch of conversions number -> date -> number ->date. I think all of the places we have dates (except for probably viewport) should start as numbers.

**Review by jmbuss — CHANGES_REQUESTED** [review]

File: packages/react-components/src/components/kpi/kpiBase.tsx (hunk 1 above)
Comment (Chandru-HM) [reply packages/react-components/src/components/kpi/kpiBase.tsx]: Done.
Now the default pattern is 'M/d/yy HH:mm:ss' and created a constant ('dd/MM/yyyy, hh:mm:ss' ) as you mentioned to use in react-components.

**Comment by Chandru-HM** [comment]
closed this PR due to many conflicts from the back merge. created the new one for the same https://github.com/awslabs/iot-app-kit/pull/2820

**Pull request closed without merging**